platform/posix/ota_pal/source/ota_pal_posix.c

```diff
@@ -263,44 +264,36 @@ static OtaPalStatus_t otaPal_CheckFileSignature( OtaFileContext_t * const C )
     EVP_PKEY * pPkey = NULL;
     EVP_MD_CTX * pSigContext = NULL;
 
-    if( C != NULL )
-    {
-        /* Extract the signer cert from the file */
-        pPkey = Openssl_GetPkeyFromCertificate( C->pCertFilepath );
+    assert( C != NULL );
 
-        /* Create a new signature context for verification purpose */
-        pSigContext = EVP_MD_CTX_new();
+    /* Extract the signer cert from the file */
+    pPkey = Openssl_GetPkeyFromCertificate( C->pCertFilepath );
 
-        if( ( pPkey != NULL ) && ( pSigContext != NULL ) )
+    /* Create a new signature context for verification purpose */
+    pSigContext = EVP_MD_CTX_new();
+
+    if( ( pPkey != NULL ) && ( pSigContext != NULL ) )
+    {
+        /* Verify an ECDSA-SHA256 signature. */
```

Comment:
What about this code is specific to `ECDSA-SHA256`?  If the answer is "nothing", then change this comment to `/* Verify the signature. */`.

Reply:
Thanks, I'll update this.